version.go

```diff
@@ -1,12 +1,13 @@
 package containerd
 
 var (
+	// Package is filled at linking time
 	Package = "github.com/containerd/containerd"
 
-	// Version holds the complete version number.
-	Version = "1.0-dev+unknown"
```

Comment:
Leave the `unknown` here. It signifies that the linker didn't actually fill in the version string and the that the version is unknown.